### PR TITLE
8341453: java/awt/a11y/AccessibleJTableTest.java fails in some cases where the test tables are not visible

### PR DESCRIPTION
**Pull request by manukumarvs**
This issue can be fixed by setting the preferred scrollable viewport size of the two tables using
        JTable table = new JTable(data, columnNames);
    +  table.setPreferredScrollableViewportSize(table.getPreferredSize());
        JTable secondTable = new JTable(data, columnNames);
    +  secondTable.setPreferredScrollableViewportSize(secondTable.getPreferredSize());
    
    The other changes in imports section are just re-arrangements and expansion of imports.

Testing: This is a manual test and it is tested and verified manually in my local system
Screenshots of before and after the fix are attached in the bug.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8341453](https://bugs.openjdk.org/browse/JDK-8341453): java/awt/a11y/AccessibleJTableTest.java fails in some cases where the test tables are not visible (**Bug** - P4)

### Reviewers
 * [Artem Semenov](https://openjdk.org/census#asemenov) (@savoptik - Committer) Review applies to [d4d83388](https://git.openjdk.org/jdk/pull/21549/files/d4d8338895abe248b665d61a31f00d39b64bc663)
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21549/head:pull/21549` \
`$ git checkout pull/21549`

Update a local copy of the PR: \
`$ git checkout pull/21549` \
`$ git pull https://git.openjdk.org/jdk.git pull/21549/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21549`

View PR using the GUI difftool: \
`$ git pr show -t 21549`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21549.diff">https://git.openjdk.org/jdk/pull/21549.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21549#issuecomment-2417628930)

### CODE REVIEW

**Comment by prrace** [comment]
"+ "Using mouse drag the header of the last culumn so the last column becomes the first one.""

culumn -> column 

And would it be better to use PassFailJFrame for this test ?

**Comment by manukumarvs** [comment]
> * JDK-8341453

@prrace Yes, it is better to use PassFailJFrame for this test and I have created a bug for the conversion https://bugs.openjdk.org/browse/JDK-8342663. I will work on this. I'm not mixing it with this fix as converting this test alone to use PassFailJFrame may not be feasible as all the tests inside the folder java/awt/a11y has a common abstract parent class AccessibleComponentTest, so it will be better to do the conversion for the whole set of tests inside java/awt/a11y folder together.

**Review by savoptik — APPROVED** [review]
LGTM

File: test/jdk/java/awt/a11y/AccessibleJTableTest.java [hunk 1]
```diff
@@ -31,13 +31,15 @@
...
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.SwingUtilities;
 import javax.swing.table.AbstractTableModel;
-import javax.swing.table.TableModel;
 
-import java.awt.*;
+import java.awt.FlowLayout;
+import java.awt.GridLayout;
```
Comment (kumarabhi006) [inline test/jdk/java/awt/a11y/AccessibleJTableTest.java:42]: Usually we put `awt imports` before swing.

**Comment by kumarabhi006** [comment]
@manukumarvs copyright year need update?
I think`@author` tag also is not required.

**Review by kumarabhi006 — APPROVED** [review]
LGTM.

**Comment by manukumarvs** [comment]
Any sponsors, please?

**Pull request closed without merging**